src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineParser.java

```diff
@@ -898,6 +960,29 @@ public String toCommandLineString(){
             }
         }
 
+        /**
+         * @param argumentDefinitionMap a map from argument names to {@link ArgumentDefinition}
+         * @return the set of arguments that this argument is mutually exclusive with (including itself in the set)
+         */
+        public Set<ArgumentDefinition> getMutuallyExclusiveArguments(Map<String,ArgumentDefinition> argumentDefinitionMap){
+            final Set<ArgumentDefinition> mutexArguments = mutuallyExclusive.stream().map(argumentDefinitionMap::get).collect(Collectors.toSet());
+            mutexArguments.add(this);
+            return mutexArguments;
+        }
+
+        public String composeMissingArgumentMessage(Map<String,ArgumentDefinition> argumentDefinitionMap){
+            if(mutuallyExclusive.isEmpty()){
+                return String.format("required argument --%s was not specified", getLongName());
```

Comment:
hmm, none of those is great because people would think they can specify multiple arguments like `--Input Foo Bar Baz`, which they can't, right? How about
`[--Input, -I : String]+` to indicate that you can specify 1 or more times?